src/azure-cli/azure/cli/command_modules/storage/operations/logging.py

```diff
@@ -15,7 +15,14 @@ def set_logging(client, log, retention, timeout=None, version=None):
 
 
 def get_logging(client, timeout=None):
+    from azure.common import AzureException
+    from knack.util import CLIError
     results = {}
     for s in client:
-        results[s.name] = s.get_logging(timeout)
+        try:
+            results[s.name] = s.get_logging(timeout)
+        except (KeyError, AzureException):
```

Comment:
what is the error code for AzureException? What if it is other error like 500

Reply:
Good catch. For service error like 500, I think it should be CloudError which is from ClientException not AzureException. 
Here what I want to catch is ConnectionError, but Storage SDK wraps it to AzureException without error code.

Reply:
added condition for ex.args type,  which should be MaxRetryError